apps/authentication-service/src/shared/http/httpService.ts

```diff
@@ -0,0 +1,142 @@
+import { ResponseError } from "auth0";
+import fetchIntercept, { FetchInterceptorResponse } from "fetch-intercept";
+
+import { AuthService, authEndpoint } from "@/shared/auth/authService";
+import { IAuthService } from "@/shared/auth/authModel";
+import {
+  IHttpDataRequest,
+  IHttpRefreshCallback,
+  IHttpRequestOptions,
+  IHttpService,
+} from "@/shared/http/httpModel";
+import { ISingleton } from "@/shared/models";
+
+const getRelativeUrl = (url: string): string => {
+  const _url = new URL(url);
+
+  return url.replace(_url.origin, "");
+};
+
+export const parseResponse = async (response: Response) => {
```

Comment:
I wouldn't expect parseResponse to live in httpService. This seems more like a shared fetching utility

Reply:
This function also needs tests

Reply:
I moved the `parseResponse` method to a share utils file